libs/utils/utils.js

```diff
@@ -109,6 +109,8 @@ export const [setConfig, getConfig] = (() => {
       config.codeRoot = conf.codeRoot ? `${origin}${conf.codeRoot}` : origin;
       config.locale = getLocale(conf.locales);
       document.documentElement.setAttribute('lang', config.locale.ietf);
+      config.direction = (new Intl.Locale(config.locale.ietf)).textInfo.direction;
```

Comment:
Should there be a check for missing/undefined config.locale.ietf?

Reply:
@chrischrischris thx for checking. Your right, if its missing or an unkown locale it throws an exception. I added a try/catch and log an error in the console. hope thats the rigth way to do it Milo.